packages/storybook/config/storybook/preview-head.html

```diff
@@ -13,7 +13,7 @@
     body {
         font-feature-settings: 'tnum'; /* Enable tabular numbers */
     }
-
+    
```

Comment:
If you fix the hex code casing above, might as well remove this whitespace change to leave the history so this file hasn't been affected by this changeset

Reply:
yep